### PR TITLE
FDG-7697 Create AFG/Explainer GA Subnavigation Events

### PR DESCRIPTION
**Pull request by hmargy**
https://federal-spending-transparency.atlassian.net/browse/FDG-7697

coverage: 89.91

### CODE REVIEW

File: src/layouts/explainer/explainer-components/explainer-sub-nav/explainer-sub-nav.jsx [hunk 1]
```diff
@@ -28,10 +28,15 @@ export default function ExplainerSubNav({hidePosition}) {
 
   const analyticsEvent = (title) => {
     Analytics.event({
-      category: 'Sitewide Navigation',
-      action: `Left Nav Click`,
+      category: 'Fiscal Data - Explainers',
```
Comment (j-hertzler) [inline src/layouts/explainer/explainer-components/explainer-sub-nav/explainer-sub-nav.jsx:31]: "Fiscal Data  -" gets prefixed to the category so we will just want this value to be "Explainers"

File: src/layouts/explainer/explainer-components/mobile-explainer-sub-nav/mobile-explainer-sub-nav.jsx [hunk 2]
```diff
@@ -87,6 +88,19 @@ export default function MobileExplainerSubNav({ hidePosition, pageName = ''}) {
     }
   };
 
+  const analyticsEvent = (title) => {
+    Analytics.event({
+      category: 'Fiscal Data - Explainers',
```
Comment (j-hertzler) [inline src/layouts/explainer/explainer-components/mobile-explainer-sub-nav/mobile-explainer-sub-nav.jsx:93]: Also remove "Fiscal Data - " here

File: src/layouts/explainer/explainer-components/explainer-sub-nav/explainer-sub-nav.jsx (hunk 1 above)
Comment (hmargy) [reply src/layouts/explainer/explainer-components/explainer-sub-nav/explainer-sub-nav.jsx]: good catch! Changes are up!

**Review by j-hertzler — APPROVED** [review]
Changes look great! Nice work getting unit tests in place for the GA4 events!

**Pull request merged**